ReactNativeClient/lib/components/screens/notes.js

```diff
@@ -25,6 +25,8 @@ class NotesScreenComponent extends BaseScreenComponent {
 	constructor() {
 		super();
 
+		this.prevFolderId_ = null;
+
```

Comment:
Could you explain the logic with prevFolderId?

Reply:
The action button is only enabled when `this.props.selectedFolderId` is not null:
https://github.com/laurent22/joplin/blob/a96734f5beb61a26a0014ab442e78ce0e3d91bd8/ReactNativeClient/lib/components/screens/notes.js#L230-L232

When the user creates a new notebook this value is set to null:
https://github.com/laurent22/joplin/blob/a96734f5beb61a26a0014ab442e78ce0e3d91bd8/ReactNativeClient/lib/components/side-menu-content.js#L207-L216

If the user then goes back without creating a new note `selectedFolderId` is still null and action button is disabled. This code tracks the last non-null folderId to use as a fallback.

Reply:
Thanks for clarifying. I think prevFolderId shouldn't be needed because selectedFolderId should be valid to begin with. Does it work if you simply remove `folderId: null`? (not sure why it's there)

Reply:
Removing `folderId: null` fixes the original issue but introduces a new one -- in the same scenario (All notes - New Notebook - go back - New Notebook again) now instead of creating a new notebook Joplin offers to edit the existing one.

Reply:
![image](https://user-images.githubusercontent.com/995612/82802919-353d7500-9e6f-11ea-909d-203229c1cdf4.png)

Reply:
![new-notbook-issue](https://user-images.githubusercontent.com/995612/82803006-51d9ad00-9e6f-11ea-9ab3-ff686e011cbf.gif)